backend/src/personnel/personnel.service.ts

```diff
@@ -453,6 +457,7 @@ export class PersonnelService {
       .createQueryBuilder('personnel')
       .leftJoinAndSelect('personnel.workLocation', 'workLocation')
       .leftJoinAndSelect('personnel.homeLocation', 'homeLocation')
+      .leftJoinAndSelect('personnel.recommitment', 'recommitment')
```

Comment:
I don't love that we're getting this all year round either. Not a dealbreaker by any means though. But if you have any thoughts, feel free to change.

Reply:
We may need this if we plan to display the recommitment status throughout the year (it sounds like they want this included on the member profile for people who did not recommit so they know the reason for being in active)